Sources/TuistKit/Commands/UpCommand.swift

```diff
@@ -16,64 +16,46 @@ class UpCommand: NSObject, Command {
     /// File handler instance to interact with the file system.
     fileprivate let fileHandler: FileHandling
 
-    /// Printer instance to output information to the user.
-    fileprivate let printer: Printing
-
-    /// Graph loader instance to load the project and its dependencies.
-    fileprivate let graphLoader: GraphLoading
-
-    /// Graph up instance to print a warning if the environment is not configured.
-    fileprivate let graphUp: GraphUpping
-
     /// Path to the project directory.
     let pathArgument: OptionArgument<String>
 
+    private let setupLoader: SetupLoading
+
     // MARK: - Init
 
     /// Initializes the command with the CLI parser.
     ///
     /// - Parameter parser: CLI parser where the command should register itself.
     public required convenience init(parser: ArgumentParser) {
-        let printer = Printer()
-        let system = System()
+        let fileHandler = FileHandler()
         self.init(parser: parser,
                   fileHandler: FileHandler(),
-                  printer: printer,
-                  graphLoader: GraphLoader(),
-                  graphUp: GraphUp(printer: printer, system: system))
+                  setupLoader: SetupLoader(fileHandler: fileHandler))
     }
 
     /// Initializes the command with its arguments.
     ///
     /// - Parameters:
     ///   - parser: CLI parser where the command should register itself.
     ///   - fileHandler: File handler instance to interact with the file system.
-    ///   - printer: Printer instance to output information to the user.
-    ///   - graphLoader: Graph loader instance to load the project and its dependencies.
-    ///   - graphUp: Graph up instance to print a warning if the environment is not configured.
```

Comment:
The `setupLoader` argument is not documented.

Reply:
Ups, updated